docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -0,0 +1,69 @@
+// tag::ess[]
+
+// QUESTION: Should we add a warning here about updating to 7.13 and include a
+// link to the upgrade docs? 
+
+{ecloud} runs a hosted version of {fleet-server}. To use {fleet-server}, start
+the service:
+
+//TODO: Describe how to start the service.
```

Comment:
The planned usage will be something close to:
Start up a cloud hosted Fleet Server by adding an APM & Fleet node in the cloud customization dialog.
<img width="805" alt="APM+Fleet-note" src="https://user-images.githubusercontent.com/12970373/116158879-4618eb00-a6bd-11eb-9b5a-713a69c1c866.png">